### ISSUE TITLE
Add 256m memory profile

### ISSUE THREAD

**Issue by proxer05** [openwrt-dev/ci-ipq60xx#3]
https://github.com/bkspc/backspacewrt/commits/main/target/linux/qualcommax 
Based on mr7350 dtsi we should be able to boot ath11k mode 2 allowing stock devices to boot.
![image](https://github.com/openwrt-dev/ci-ipq60xx/assets/62423996/4daf1910-d7b4-4ec6-bd4b-104cacf158b4)
![image](https://github.com/openwrt-dev/ci-ipq60xx/assets/62423996/eb3ed085-9b11-482d-a70f-1e187029da23)
Memory usage with dns recursor (unbound). Basic functionality without services like this should be possible(as on stock fw).

**Comment by pexcn** [comment]
Do you mean that 256M of memory can also enable WiFi?

**Comment by proxer05** [comment]
Exactly. That's how Xiaomi and other OEMs are able to use 256mb dies.

**Comment by proxer05** [comment]
https://pastebin.com/raw/HTrZbvRv This should explain you everything.(from qsdk 4.4.60)
